src/config.rs

```diff
@@ -21,6 +21,8 @@ pub struct Config {
     pub username: String,
     /// The password for the email account.
     pub password: String,
+    /// The directory of the config file
+    pub directory: Option<PathBuf>,
```

Comment:
This whole approach is wrong. There is always a config file, DMS will create one using `directories-next` crate in the default `~/.config/deadman` analogous paths in Windows and MacOS.
This should not be an `Option` instead it should default to something.